.travis.yml

```diff
@@ -7,9 +7,12 @@ language: objective-c
 env:
   matrix:
     
-    - CONDA_PY=27  
-    - CONDA_PY=34  
-    - CONDA_PY=35  
+    - CONDA_NPY=110  CONDA_PY=27
```

Comment:
> Maybe we can chat about the issue I was running into and see if there was a usage error on my part or something that needs fixing.

Ah, yes. Thanks @jakirkham. I'd say it was something that could be improved with conda-smithy. You shouldn't need tokens to re-render.